app/services/articles/feed.rb

```diff
@@ -0,0 +1,43 @@
+module Articles
+  class Feed
+    attr_reader :stories
+
+    def initialize(number_of_articles:, page:, tag: nil)
+      @number_of_articles = number_of_articles
+      @page = page
+      @tag = tag
+      @stories = published_articles_by_tag
```

Comment:
should this load data on initialization?

Reply:
Hmm, good point, that's probably too aggressive